invoicer_project/clients/templates/clients/client_edit.html

```diff
@@ -4,6 +4,8 @@
 <head>
     <meta charset="UTF-8"/>
     <title>Edit client</title>
+    <script src="https://code.jquery.com/jquery-3.6.3.min.js"></script>
```

Comment:
Do you need jquery? I thought we have got rid it

Reply:
Yes, we need. New dropdown component works with jQuery. I will download jQuery locally to avoid downloading from the net

Reply:
P.S I've downloaded it